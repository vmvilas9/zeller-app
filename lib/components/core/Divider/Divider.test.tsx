import renderer from 'react-test-renderer';
import {Divider} from "./index";
import * as React from "react";

it(`renders correctly`, () => {
    const tree = renderer.create(<Divider/>).toJSON();

    expect(tree).toMatchSnapshot();
});
