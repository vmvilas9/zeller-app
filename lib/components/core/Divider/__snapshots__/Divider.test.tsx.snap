// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "#FFFFFF",
      },
      {
        "backgroundColor": "#E2E8F0",
        "height": 1,
        "marginVertical": 24,
      },
    ]
  }
/>
`;
